server/controllers/accountController.js

```diff
@@ -68,14 +68,19 @@ export default class authUsers{
                 status: 400,
                 message: 'only numbers are allowed in the account number field'
             });
-        }
+        } else {
         var AcctId = parseInt(req.params.accountNumber);
```

Comment:
Use a more descriptive variable name here.

Reply:
changed to AccountNumber